physx/src/physics.rs

```diff
@@ -711,9 +735,21 @@ impl<Allocator: AllocatorCallback> PhysicsFoundationBuilder<Allocator> {
         self
     }
 
+    /// Set error callback
+    /// # Safety
+    /// `error_callback` must live as long as the object created by `build`
```

Comment:
this safety note is no longer needed right?

Reply:
This is still a public API so I think it makes sense to annotate an unsafe function even if our preferred method is fully safe.

Reply:
but this method isn't unsafe anymore?

Reply:
Oh! Sorry! Yes. 🤦